movement/io/save_poses.py

```diff
@@ -87,11 +186,32 @@ def to_dlc_file(ds: xr.Dataset, file_path: Union[str, Path]) -> None:
     file_path : pathlib Path or str
         Path to the file to save the DLC poses to. The file extension
         must be either ".h5" (recommended) or ".csv".
+    split_individuals : bool, optional
+        Format of the DeepLabcut output file.
+        - If True, the file will be formatted as in a single-animal
+        DeepLabCut project: no "individuals" level, and each individual will be
+        saved in a separate file. The individual's name will be appended to the
+        file path, just before the file extension, i.e.
+        "/path/to/filename_individual1.h5".
+        - If False, the file will be formatted as in a multi-animal
+        DeepLabCut project: the columns will include the
+        "individuals" level and all individuals will be saved to the same file.
+        - If "auto" the format will be determined based on the number of
+        individuals in the dataset: True if there is only one, and
+        False if there are more than one. This is the default.
 
     See Also
     --------
     to_dlc_df : Convert an xarray dataset containing pose tracks into a
-        DeepLabCut-style pandas DataFrame with multi-index columns.
+    DeepLabCut-style pandas DataFrame with multi-index columns
+    for each individual or a dictionary of DataFrames for each individual
+    based on the 'split_individuals' argument.
```

Comment:
For this docstring to render correctly in Sphinx, these lines need to be indented:

```suggestion
        DeepLabCut-style pandas DataFrame with multi-index columns
        for each individual or a dictionary of DataFrames for each individual
        based on the 'split_individuals' argument.
```